src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const mainImage = document.querySelector('#largeImg');
+
+  const thumbnails = document.querySelector('#thumbs');
+
+  function changeImage(e) {
+    const necessaryLink = e.target.closest('a');
+
+    if (necessaryLink) {
+      e.preventDefault();
+
+      const clickedImage = necessaryLink.querySelector('img');
+
+      if (clickedImage) {
+        const newSrc = necessaryLink.getAttribute('href');
```

Comment:
Consider using the .href property instead of the getAttribute method. The .href property gets the href (URL) of the linked resource whereas the getAttribute method returns the value of the href attribute as it was set in the HTML code. In some cases, the two values can be different (for example, if the href attribute is a relative URL).